.travis.yml

```diff
@@ -1,27 +1,32 @@
 language: python
 python:
- - 3.6
+  - 2.7
+  - 3.6
 sudo: false
-env:
- - TOX_ENV=py27
- - TOX_ENV=py36
- - TOX_ENV=docs
- - TOX_ENV=lint
- - TOX_ENV=eslint
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=docs
+    - python: 2.7
```

Comment:
Good question. I was going to say bump linting and docs steps up to py3 when our code is ready. I don't know if bumping up before then would yield any unwarranted effects or not from the code not being quiet py3 compatible.